executor/common_bsd.h

```diff
@@ -345,13 +346,17 @@ static int do_sandbox_setuid(void)
 	initialize_tun(procid);
 #endif
 
-	const int nobody = 65534;
+	char pwbuf[1024];
+	struct passwd *pw, pwres;
+	if (getpwnam_r("nobody", &pwres, pwbuf, sizeof(pwbuf), &pw) != 0 || !pw)
```

Comment:
Actually 65534 can be anything, but it's not nobody on NetBSD.

Reply:
65533 and a comment then? Out of curiosity, which part of the kernel has that magic number baked in?

Reply:
I want to pick nobody the same as everywhere.

Reply:
If understand the intent of this code correctly, we need any unprivileged user. So, pick a number. It will be the same for everybody on *BSD. In fact, @dvyukov already picked the number, I simply copied it from his Linux code.